tests/Doctrine/Tests/ORM/Query/QueryTest.php

```diff
@@ -303,4 +304,65 @@ public function testSelectFromSubquery()
         $this->expectExceptionMessage('Subquery');
         $query->getSQL();
     }
+
+    /**
+     * @group #6699
```

Comment:
We're using the ticket id without the `#` everywhere else, we should keep it consistent here too